### PR TITLE
Port helpers.js to polyfills.js

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Removed all the helper functions from helpers
- Renamed helpers.js to polyfills.js as it only contains a polyfill
- Removed the A/B test on the contribution page

## QA
- Load the site and check there are no js errors
- Go to `http://localhost:8001/download/desktop/contribute?version=16.04.1&architecture=amd64` and check the title loads ok and there are no errors in the console
- Check for errors on: `/cloud/openstack/managed-cloud/` as the helpers import was removed

### CODE REVIEW

File: templates/download/desktop/contribute.html [hunk 1]
```diff
@@ -15,10 +15,6 @@
 <div id="contribute-introductions" class="row row-hero contribution-intro-row">
     <div class="strip-inner-wrapper">
         <div class="contribution-intro nine-col">
-            <h1>Tell us what we should do more&hellip;</h1>
-            <p>&hellip;and put your money where your mouth is ;)</p>
-        </div>
-        <div class="contribution-intro nine-col">
             <h1>Help shape the future of Ubuntu&hellip;</h1>
             <p>&hellip;tell us what to do more of!</p>
```
Comment (nottrobin) [inline templates/download/desktop/contribute.html]: Do you think we could find a way to remove the "of!"? Maybe simply remove it and leave "Help shape the future of Ubuntu"? Or make the second line "What should we focus on?" or "Tell us what's most important to you."

File: templates/download/desktop/contribute.html [hunk 2]
```diff
@@ -15,10 +15,6 @@
 <div id="contribute-introductions" class="row row-hero contribution-intro-row">
     <div class="strip-inner-wrapper">
         <div class="contribution-intro nine-col">
-            <h1>Tell us what we should do more&hellip;</h1>
```
Comment (nottrobin) [inline templates/download/desktop/contribute.html]: :heart: love the idea of not bother with the silly random introduction

File: templates/download/desktop/thank-you.html [hunk 3]
```diff
@@ -86,7 +86,7 @@
 {% endblock content %}
 
 {% block footer_extra %}
-<script src="{% versioned_static 'js/helpers.js' %}"></script>
+<script src="{% versioned_static 'js/ployfills.js' %}"></script>
```
Comment (nottrobin) [inline templates/download/desktop/thank-you.html]: Typo breaks this page. Should be "polyfills.js".

File: templates/download/server/thank-you.html [hunk 4]
```diff
@@ -59,7 +59,7 @@
 
 {% endblock content %}
 {% block footer_extra %}
-<script src="{% versioned_static 'js/helpers.js' %}"></script>
+<script src="{% versioned_static 'js/ployfills.js' %}"></script>
```
Comment (nottrobin) [inline templates/download/server/thank-you.html]: Typo breaks this page. Should be "polyfills.js".

**Comment by nottrobin** [comment]
2 essential fixes, one wording suggestion.

Once those are taken care of it all looks good.

File: templates/download/desktop/contribute.html (hunk 1 above)
Comment (pmahnke) [inline templates/download/desktop/contribute.html]: +1 to Tell us what is most important to you.

File: templates/download/desktop/thank-you.html (hunk 3 above)
Comment (anthonydillon) [inline templates/download/desktop/thank-you.html]: Good spot, thanks.

File: templates/download/server/thank-you.html (hunk 4 above)
Comment (anthonydillon) [inline templates/download/server/thank-you.html]: Good spot, thanks.

File: templates/download/desktop/contribute.html (hunk 1 above)
Comment (anthonydillon) [inline templates/download/desktop/contribute.html]: Updated.

**Comment by anthonydillon** [comment]
Thanks @nottrobin, updated with your review points.

**Comment by nottrobin** [comment]
Thanks for doing those @anthonydillon, LGTM :+1:

**Pull request merged**